srcpkgs/webhook/files/webhook/run

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+exec 2>&1
+
+[ -r conf ] && . ./conf
+
+OPTS=""
+
+[ "$TEMPLATE" = "true" ] && OPTS="$OPTS -template"
+[ "$X_REQUEST_ID" = "true" ] && OPTS="$OPTS -x-request-id"
+[ "$SECURE" = "true" ] && OPTS="$OPTS -secure"
+[ "$NOPANIC" = "true" ] && OPTS="$OPTS -nopanic"
+[ "$DEBUG" = "true" ] && OPTS="$OPTS -debug"
+
+if [ "$HOTRELOAD" = "true" ] || [ -z "$HOTRELOAD" ]; then OPTS="$OPTS -hotreload"; fi
+if [ "$VERBOSE" = "true" ] || [ -z "$VERBOSE" ]; then OPTS="$OPTS -verbose"; fi
+
+[ -n "$IP" ] && OPTS="$OPTS -ip $IP"
+[ -n "$PORT" ] && OPTS="$OPTS -port $PORT"
+[ -n "$PREFIX" ] && OPTS="$OPTS -urlprefix $PREFIX"
+[ -n "$HTTP_METHODS" ] && OPTS="$OPTS -http-methods $HTTP_METHODS"
+[ -n "$X_REQUEST_ID_LIMIT" ] && OPTS="$OPTS -x-request-id-limit $X_REQUEST_ID_LIMIT"
+[ -n "$MAX_MULTIPART_MEM" ] && OPTS="$OPTS -max-multipart-mem $MAX_MULTIPART_MEM"
+[ -n "$CERT" ] && OPTS="$OPTS -cert $CERT"
+[ -n "$KEY" ] && OPTS="$OPTS -key $KEY"
+[ -n "$CIPHER_SUITES" ] && OPTS="$OPTS -cipher-suites $CIPHER_SUITES"
+[ -n "$TLS_MIN_VERSION" ] && OPTS="$OPTS -tls-min-version $TLS_MIN_VERSION"
+
+[ -z "$HOOKS_FILES" ] && HOOKS_FILES="/etc/webhook/hooks.json"
+for f in $HOOKS_FILES; do
+    OPTS="$OPTS -hooks $f"
+done
+
+for h in $HEADERS; do
+    OPTS="$OPTS -header $h"
+done
+
+exec webhook $OPTS $OTHER_OPTS -pidfile /run/webhook.pid
```

Comment:
Why do you need to specify a PID file?

Reply:
this was suggested in #xbps. by specifying the option, webhook manages the PID file itself.